tests/testthat/test-core_loaders.R

```diff
@@ -0,0 +1,193 @@
+test_that("load_file with a .csv works", {
+  skip_if_not_installed("readr")
+
+  file <- withr::local_file(tempfile(fileext = ".csv"))
+
+  # create a CSV file
+  df <- tibble::tibble(a = 1L:3L, b = c("a", "b", "c"))
+  readr::write_csv(df, file)
+
+  # load the CSV file
+  loaded_df <- suppressMessages(load_file(file))
```

Comment:
If you want to avoid the message about column types you can set `show_col_types=FALSE`